[["/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/__init__.pyi",[false,[["HttpLib2Error",true,2,["HttpLib2Error","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["HttpLib2ErrorWithResponse",true,2,["HttpLib2ErrorWithResponse","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["RedirectMissingLocation",true,2,["RedirectMissingLocation","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["RedirectLimit",true,2,["RedirectLimit","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["FailedToDecompressContent",true,2,["FailedToDecompressContent","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["UnimplementedDigestAuthOptionError",true,2,["UnimplementedDigestAuthOptionError","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["UnimplementedHmacDigestAuthOptionError",true,2,["UnimplementedHmacDigestAuthOptionError","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["MalformedHeader",true,2,["MalformedHeader","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["RelativeURIError",true,2,["RelativeURIError","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["ServerNotFoundError",true,2,["ServerNotFoundError","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["ProxiesUnavailableError",true,2,["ProxiesUnavailableError","/home/user/.vscode-server/extensions/ms-python.vscode-pylance-2023.11.10/dist/typeshed-fallback/stubs/httplib2/httplib2/error.pyi",5]],["__author__",true,13,null],["__copyright__",true,13,null],["__contributors__",true,13,null],["__license__",true,13,null],["__version__",true,13,null],["debuglevel",true,13,null],["RETRIES",true,14,null],["Authentication",true,5,null],["BasicAuthentication",true,5,null],["DigestAuthentication",true,5,null],["HmacDigestAuthentication",true,5,null],["WsseAuthentication",true,5,null],["GoogleLoginAuthentication",true,5,null],["FileCache",true,5,null],["Credentials",true,5,null],["KeyCerts",true,5,null],["AllHosts",true,5,null],["ProxyInfo",true,5,null],["HTTPConnectionWithTimeout",true,5,null],["HTTPSConnectionWithTimeout",true,5,null],["Http",true,5,null],["Response",true,5,null]]]]]